server/core/src/main/java/org/infinispan/server/core/telemetry/inmemory/InMemoryTelemetryClient.java

```diff
@@ -0,0 +1,50 @@
+package org.infinispan.server.core.telemetry.inmemory;
+
+import java.util.List;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.StatusCode;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.sdk.trace.data.SpanData;
+
+public class InMemoryTelemetryClient {
```

Comment:
It looks like this is only used for tests. It should be moved to the test sources.